[2024-10-09T00:00:08.336+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-09T00:00:08.364+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: airbyte_sync_tables_postgres.airbyte_check_sync scheduled__2024-10-08T00:00:00+00:00 [queued]>
[2024-10-09T00:00:08.384+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: airbyte_sync_tables_postgres.airbyte_check_sync scheduled__2024-10-08T00:00:00+00:00 [queued]>
[2024-10-09T00:00:08.386+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-09T00:00:08.423+0000] {taskinstance.py:2330} INFO - Executing <Task(AirbyteJobSensor): airbyte_check_sync> on 2024-10-08 00:00:00+00:00
[2024-10-09T00:00:08.436+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'airbyte_sync_tables_postgres', 'airbyte_check_sync', 'scheduled__2024-10-08T00:00:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/airbyte_sync_tables_postgres.py', '--cfg-path', '/tmp/tmprdybufez']
[2024-10-09T00:00:08.441+0000] {standard_task_runner.py:91} INFO - Job 17: Subtask airbyte_check_sync
[2024-10-09T00:00:08.442+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=2753) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-09T00:00:08.444+0000] {standard_task_runner.py:64} INFO - Started process 2754 to run task
[2024-10-09T00:00:08.542+0000] {task_command.py:426} INFO - Running <TaskInstance: airbyte_sync_tables_postgres.airbyte_check_sync scheduled__2024-10-08T00:00:00+00:00 [running]> on host efbb3a47dac9
[2024-10-09T00:00:08.840+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='vizentec-engdados' AIRFLOW_CTX_DAG_ID='airbyte_sync_tables_postgres' AIRFLOW_CTX_TASK_ID='airbyte_check_sync' AIRFLOW_CTX_EXECUTION_DATE='2024-10-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-08T00:00:00+00:00'
[2024-10-09T00:00:08.844+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-09T00:00:08.915+0000] {baseoperator.py:400} WARNING - AirbyteJobSensor.execute cannot be called outside TaskInstance!
[2024-10-09T00:00:08.940+0000] {base.py:84} INFO - Using connection ID 'airbyte_conn' for task execution.
[2024-10-09T00:00:09.126+0000] {airbyte.py:113} INFO - Waiting for job 20 to complete.
[2024-10-09T00:01:09.574+0000] {base.py:84} INFO - Using connection ID 'airbyte_conn' for task execution.
[2024-10-09T00:01:09.964+0000] {airbyte.py:113} INFO - Waiting for job 20 to complete.
[2024-10-09T00:02:10.004+0000] {base.py:84} INFO - Using connection ID 'airbyte_conn' for task execution.
[2024-10-09T00:02:10.538+0000] {airbyte.py:108} INFO - Job 20 completed successfully.
[2024-10-09T00:02:10.539+0000] {base.py:294} INFO - Success criteria met. Exiting.
[2024-10-09T00:02:10.541+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-09T00:02:10.567+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=airbyte_sync_tables_postgres, task_id=airbyte_check_sync, run_id=scheduled__2024-10-08T00:00:00+00:00, execution_date=20241008T000000, start_date=20241009T000008, end_date=20241009T000210
[2024-10-09T00:02:10.641+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-10-09T00:02:10.714+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-09T00:02:10.718+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
